lib/internal/Magento/Framework/Image/Adapter/ImageMagick.php

```diff
@@ -5,6 +5,11 @@
  */
 namespace Magento\Framework\Image\Adapter;
 
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Image adapter from ImageMagick
+ */
 class ImageMagick extends \Magento\Framework\Image\Adapter\AbstractAdapter
```

Comment:
~Import class.~

Actually it's in the same namespace, so you can directly use AbstractAdapter without full path.